java/src/jmri/jmrix/dccpp/serial/SerialDCCppPacketizer.java

```diff
@@ -68,7 +68,7 @@ protected int lengthOfByteStream(final jmri.jmrix.AbstractMRMessage m) {
      */
     private boolean backgroundRefreshStarted = false;
 
-    final class RefreshAction implements ThreadingUtil.ThreadAction {
+    final class RefreshAction implements Runnable {
```

Comment:
Could you comment on why you changed `ThreadingUtil.ThreadAction` to `Runnable` here?

Reply:
I am looking at allowing TLS protected communications because Apple's language keeps hinting that eventually they will bar non-protected communications on iPhones, and the techniques I've been looking at sometimes produce a Runnable as a method result, and I'd like to just pass that in instead of wrapping a Runnable inside a Runnable (because I really can't think of any reason to do that unless the outer Runnable also implements something else like Future that allows its behavior when running to be changed).